src/oc/ocWrapper.ts

```diff
@@ -437,6 +439,112 @@ export class Oc {
         return result.stdout;
     }
 
+    public async deleteProject(projectName: string): Promise<string> {
+        const obj = await this.isOpenShiftCluster() ? 'project' : 'namespace';
+        return await CliChannel.getInstance().executeTool(
+                new CommandText('oc', `delete ${obj} ${projectName}`)
+            )
+            .then((result) => result.stdout)
+            .catch((error) => error.message);
```

Comment:
ACK

Reply:
@datho7561  Hm. Actually it's not correct. Previously we were using `-f` parameter when running `odo` commands which was preventing it from throwing any exceptions (f.i. when deleting non-existing projects).
Catching errors when an  `oc` command is used makes it work as`odo xxx -f` did before, otherwise we'll have to surround every call to `Oc.Instance.deleteProject()` with a try-catch or errors in some other way.

Reply:
I agree that it will probably change some behaviour, but I think it should be okay. We should avoid presenting the option to delete a project that doesn't exist in the UI. Even if you attempted to, it would still display the error as a popup notification. (To confirm this, try deleting the `default` project on a `kind` cluster)